src/bindings/g_object/enum.cr

```diff
@@ -35,8 +35,13 @@ abstract struct Enum
     @@_g_type
   end
 
+  # :nodoc:
+  def self._is_flags_enum? : Bool
+    {{ !!@type.annotation(Flags) }}
+  end
```

Comment:
This could be `GICrystal::flag?(obj)` instead.

Reply:
No, in the GValue implementation we need to find out if a given class object is a flags enum (`value.class._is_flags_enum?`), but we can't access annotations at runtime